### PR TITLE
Use github actions for both ros1 and ros2

### PR DESCRIPTION
**Pull request by Dale-Koenig**
Use github actions for both ros builds separately, and change build method.  
Coverity scan may not work but I need the action to be merged once to test

### CODE REVIEW

File: .github/workflows/ros1_build_test/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# syntax=docker/dockerfile:1.4
+
+FROM ros:noetic
+SHELL ["/bin/bash", "-c"]
+
+RUN <<sss
+  pwd
+  ls
+sss
```
Comment (Dale-Koenig) [inline .github/workflows/ros1_build_test/Dockerfile]: Remove

File: .github/workflows/ros1_build_test/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  apt-get update
+  apt-get install -y python3-catkin-tools
+  rosdep install -y --from-paths . --ignore-src --rosdistro noetic
+deps
+
+RUN <<build
+  cd /root/catkin_ws
+  source /opt/ros/noetic/setup.bash
+  catkin init
+  catkin config --skiplist alica_tracing --cmake-args -DCMAKE_CXX_FLAGS=-Werror
```
Comment (triccyx) [inline .github/workflows/ros1_build_test/Dockerfile]: -Werror should be already present in CMake as default flag

Comment (Dale-Koenig) [reply .github/workflows/ros1_build_test/Dockerfile]: Got it, this was just copied but will remove here

**Review by dhananjay-patki — APPROVED** [review]

File: .github/workflows/ros1_build_test/Dockerfile (hunk 2 above)
Comment (Dale-Koenig) [inline .github/workflows/ros1_build_test/Dockerfile]:
```suggestion
  catkin config --skiplist alica_tracing
```

**Review by triccyx — APPROVED** [review]

**Pull request merged**